Patch that has changes after autoreconf -fi.
==========================================================================
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -72,7 +72,8 @@
 	"$(DESTDIR)$(libvdpauincludedir)"
 LTLIBRARIES = $(lib_LTLIBRARIES)
 am__DEPENDENCIES_1 =
-libvdpau_la_DEPENDENCIES = $(am__DEPENDENCIES_1) $(am__DEPENDENCIES_1)
+libvdpau_la_DEPENDENCIES = $(am__DEPENDENCIES_1) $(am__DEPENDENCIES_1) \
+	$(am__DEPENDENCIES_1)
 am__libvdpau_la_SOURCES_DIST = vdpau_wrapper.c mesa_dri2.c mesa_dri2.h
 @DRI2_TRUE@am__objects_1 = mesa_dri2.lo
 am_libvdpau_la_OBJECTS = vdpau_wrapper.lo $(am__objects_1)
@@ -249,7 +250,8 @@
 
 libvdpau_la_LIBADD = \
     $(DLOPEN_LIBS) \
-    $(XEXT_LIBS)
+    $(XEXT_LIBS) \
+    $(X11_LIBS)
 
 libvdpau_la_LDFLAGS = -version-info 1:0:0 -no-undefined
 libvdpauincludedir = $(includedir)/vdpau
